src/appengine/handlers/corpora.py

```diff
@@ -48,6 +48,7 @@ class CreateHandler(base_handler.Handler):
   """Create a corpus."""
 
   @handler.check_user_access(need_privileged_access=True)
+  @handler.post(handler.FORM, handler.HTML)
```

Comment:
Not directly relevant to this PR:
Does this decorator need to be added to every `post` handler? 
Couldn't this be done in a less repetitive way?

Reply:
Probably, but I can't think of anything obvious that would completely remove the issue. We'd still need to provide the content type information somehow, so even if we did it without the decorator we'd sill need to set it somewhere. It's definitely worth considering though since this seems fairly error-prone.